pom.xml

```diff
@@ -125,22 +125,16 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-pmd-plugin</artifactId>
 				<version>3.13.0</version>
-				<executions>
-					<execution>
-						<id>pmd</id>
-						<phase>verify</phase>
-						<goals>
-							<goal>check</goal>
-						</goals>
-						<configuration>
-							<failurePriority>2</failurePriority>
-							<failOnViolation>true</failOnViolation>
-							<rulesets>
-								<ruleset>configs/pmd-rules.xml</ruleset>
-							</rulesets>
-						</configuration>
-					</execution>
-				</executions>
+				<goals>
```

Comment:
I just checked, and found that this change disabled the PMD checks. Could you please undo this change and fix the PMD warnings?

Reply:
Actually IDEA showed errors about <rulesets> section. I'll revert it for now.

Reply:
Okay, I see. This also ensures that the Travis CI gate only passes when no PMD warnings are issued.